### ISSUE TITLE
[Feature Request]  Plot Extra Button

### ISSUE THREAD

**Issue by starman2908** [Tikipeter/repository.tikipeter#288]
Hi Peter, hope this message finds you well.

Brilliant work on the extras customisation options. Just wondering if it would be possible to add an extras button for Plot (Extras>Set actions for extras buttons) that would display the dialogue plot box when pressed.

Cheers

**Comment by Tikipeter** [comment]
I don't understand.

If you click on the plot, a dialog plot box will be shown.

Do you want to have the plot hidden, and then have a button for it?

**Comment by starman2908** [comment]
Yeah that's exactly what I meant, apologies I should have been more specific. I generally hide the plot in the extras dialogue as usually have plot displayed through the skin because the fonts are bigger for me to read more easily (getting older and blinder🤣)  
Your extras dialogue beats any info dialogue I've used in any skin and when clicking through to other content using the extras dialogue, would be handy if there was a plot button, so I could bring up the plot dialogue as it preserves the larger font I use in the skin.

No worries if it's too much trouble, I can always unhide the plot box and click on it for the same effect. It was only redundant to me because I had already read the plot on my skin's view, but then it dawned on me if I browse and click anything using extras there'd also be no plot box and a extras button would be so handy 👍

**Comment by Tikipeter** [comment]
I understand.

I'll have a look when I'm home from work. I haven't touched that code in a while, but it should be possible.

**Comment by Tikipeter** [comment]
Done for next update.

**Comment by starman2908** [comment]
You sir, are a legend! Thank you

**Comment by starman2908** [comment]
I've installed the latest from the test repository. Thank you very much for the plot button in the action button options. I've run into an issue opening extras and wanted to give a heads up.  After unchecking plot in the >Enable Content for Extras Lists... i'm getting "Fen Error" dialogue on trying to launch, are you able to replicate the behaviour?

**Comment by les017** [comment]
I found the same error after test report update. Ultimately I went Tools/Settings/Features/Restore Extras Buttons to Default Actions.

Then had to rebuild my custom Extras content. No problem since then.

**Comment by Tikipeter** [comment]
The code that's going to run next update to sort this out won't run on incremental updates like this. So les017's solution is the  only option.

**Comment by starman2908** [comment]
Ah brilliant, thanks for the info and suggestion guys :)
